plugins/fancynpcs/src/main/java/de/oliver/fancynpcs/skins/SkinManagerImpl.java

```diff
@@ -64,8 +65,8 @@ public SkinData getByIdentifier(String identifier, SkinData.SkinVariant variant)
         if (SkinUtils.isPlaceholder(identifier)) {
             String parsed = ChatColorHandler.translate(identifier);
 
-            if (SkinUtils.isPlaceholder(parsed)) {
-                return null;
+            if (parsed.isBlank() || parsed.equalsIgnoreCase("null") || SkinUtils.isPlaceholder(parsed)) {
```

Comment:
This is needed because placeholders like `%player_name%` returns an empty string when parsed with no player specified. Seen `null` in some other expansion but forgot which one.